fastly/block_fastly_service_product_enablement.go

```diff
@@ -535,7 +535,7 @@ func (h *ProductEnablementServiceAttributeHandler) checkAPIError(err error) erro
 	if he, ok := err.(*gofastly.HTTPError); ok {
 		if he.StatusCode == http.StatusBadRequest {
 			for _, e := range he.Errors {
-				if strings.Contains(e.Title, "is not entitled to disable product") {
+				if strings.Contains(e.Title, "not entitled to disable") || strings.Contains(e.Title, "product cannot be disabled") {
```

Comment:
Two different users:

1. Doesn't have access to the product
2. Has access but not the permission to disable it.